### ISSUE TITLE
[FEATURE] Multicard title etc

### ISSUE THREAD

**Issue by pilpfiction** [mennovanhout/flutter-hdashboards-issues#131]
**Is your feature request related to a problem? Please describe.**
Yes, I can't change the capability name/label anymore since v0.44.

Previously I was able to set a title for a capability while adding it to a multicard. This gave me the opportunity to freely name the label displayed in the multicard. Also only one single line was displayed - which is beneficial while using HD on a mobile device.

Since v0.44 always two lines are displayed for each capability. Resulting in two issues:

1. Now a multicard requires more vertical space 
2. Additional line of information has no real value in my scenario (especially while I am now also able to add additional sub titles)!

**Describe the solution you'd like**
1. Make the title optionally while adding the capability
2. Allow to change the capability name/label as well

One reason to do so:
I have an AWHP from LG and I use the LG ThinQ App in Homey to operate the heater. The app has capabilities with terms like "water in",  "water out" and "power usage". The first two options are correct, nevertheless I would like to change these to "Vorlauf" and "Rücklauf", which are terms used outside of the homey world. The latter term is even not translated and quite long, thus I would like to be able to change it in HD as well.

Until version 0.43 I was able to achieve that, as mentioned before, but now with 0.44 I am not able to change those anymore.

**Describe alternatives you've considered**
Create a virtual device and use capability labels I want to be displayed in HD

**Additional context**
I tried to summarize what has been, currently is and shall be possible in regards to title and labe when adding capabilities to a multicard;

![Screenshot_20231212-110018](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/71706fd7-70bb-4513-a0e9-b0c554aecf9b)
![Screenshot_20231212-110035](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/78ff179e-a778-4511-9690-83ed5d96aea1)
![Screenshot_20231212-110057](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/bc3d4eb2-2f7d-4201-b19e-0f2ffd7f33bd)

Here as it currently works in HD:
![Screenshot_20231212-111713](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/445798b6-0690-4669-aec0-d966ac7198e3)

In this screen you can see that 
1. The title for the capabilities ("+0" and "+1") are not helpful/necessary. It would be great to be able to drop those (you already mentioned to make this optionally with the next update).
2. The app YR used has also capabilities which uses englisch terms and are long. It would be great when I could change those as well. In this case just to "Luftdruck".

Resulting in this 
![Screenshot_20231212-111713~2](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/472bd60b-18db-4da6-8e2c-72dd5e1e9253)

**Comment by mennovanhout** [comment]
Haha now i also understand our confusion in Discord.

The title in multitile overrides the **Device name**.
It has nothing to do at all with capabilities.
So I didn't understand what you were meaning with this.

This update added **Capability title** as this did NOT exist in multi tiles, only device name / custom device name did.
But this also leaves me to the conclusion:

Your request is indeed to **remove** the **capability** title. This will leave the device name / custom title as it was in v0.4.3?

**Comment by pilpfiction** [comment]
Not sure, because when you remove the capability title, then I have no option to rename the capability name/label as it was in v0.43, correct?

As I remember correctly, in v0.43 I was able to enter a title in the capability I am just about to add. HD prefilled this capability title field with the device name, not the label/name of the capability. But this gave me the option to enter anything I wanted. This way I was able to enter any free text for the capability. 

And this is what I would like to be able again.

It is really bit confusing as I might not use the right terms for each function/feature in HD as you understand them.

In my view I indeed don't necessarily need a capability title (whereas in other scenarios it might be useful) when adding such a capability to a multicard. I just want the capability name/label displayed (single line) **and** be able to change it.

Hope this helps to understand my thoughts!
